ui/desktop/app/components/settings-card/server/index.hbs

```diff
@@ -0,0 +1,41 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<SettingsCard
+  @header={{t 'settings.server.title'}}
+  @icon='cloud'
+  @description={{t 'settings.server.description'}}
+>
+  <:state>
+    {{#if @model.serverInformation}}
+      <Hds::Badge
+        @text={{t 'settings.connected'}}
+        @icon='check'
+        @color='success'
+      />
+    {{/if}}
```

Comment:
Im not sure what the intended functionality is, but when i shut down my self-managed instance the "Connected" badge does not disappear. 

https://github.com/user-attachments/assets/a2871fb2-4fcb-4960-84ea-30c73a88002f

Reply:
hmmm good catch, so right now I'm just using the cluster URL to display the `connected`, I'll check what we do in other pages to throw the error and display disconnected. 

Also, now I'm wondering why there is no error in the application block

Reply:
What would error out in the application block? Nothing there would be hitting the controller I believe. I'm not sure if this is one that matters too much if I were to be honest since it likely would occur only locally since you have to bring down the controller

Reply:
yeah, that is right unless we refresh the view, I don't see a way to check if the controller is down.   

should we remove the `connected` word to avoid any confusion?

Reply:
good point, I will leave this as is for now. But will check with design/PM and include this for our polishing work if the decision is to add a button

Reply:
checked with design, we will be removing the label `connected` to avoid any confusion.  added a separate ticket for this